// api/mockGoods.js
export default [
    {
        id: '1',
        categoryId: '1',
        name: 'Виагра, 100 мг, 2 шт.',
        description:
            'Лечение нарушений эрекции, характеризующихся неспособностью к достижению или сохранению эрекции полового члена, достаточной для удовлетворительного полового акта.',
        price: 823,
        image: '/uploads/viagra10mg2.jpg',
        manufacturer: 'Производитель',
        substance: 'Действующее вещество',
        shelf_life: 'Срок годности',
        pharmachologic: '1',
        effect: '',
        indications: '',
        contraindications: '',
        side_effect: ''
    },
    {
        id: '2',
        categoryId: '1',
        name: 'Виагра, 100 мг, 1 шт.',
        description:
            'Лечение нарушений эрекции, характеризующихся неспособностью к достижению или сохранению эрекции полового члена, достаточной для удовлетворительного полового акта.',
        price: 953,
        image: '/uploads/1.jpg',
        manufacturer: 'Пфайзер, США',
        substance: 'Силденафил',
        shelf_life: 'Длинный срок',
        pharmachologic: '1',
        effect: 'Происходит расслабление гладких мышц и усиление кровотока в кавернозном теле.',
        indications: 'Лечение нарушений эрекции, характеризующихся неспособностью к достижению или сохранению эрекции полового члена, достаточной для удовлетворительного полового акта.',
        contraindications: 'Повышенная чувствительность к любому компоненту препарата.',
        side_effect: 'Нежелательные явления обычно преходящие и легкие или умеренно выраженные. Частота нежелательных явлений повышается с увеличением дозы.'
    },
    {
        id: '3',
        categoryId: '4',
        name: 'Зубная паста Ajona, 25 мл',
        description:
            '',
        price: 128,
        image: '/uploads/2.jpeg',
        manufacturer: 'Доктор Р. Либе, Германия',
        substance: 'Вода, кальция карбонат, глицерин',
        shelf_life: 'Длинный срок',
        pharmachologic: '1',
        effect: '132 grams (4.7 ounces) with battery',
        indications: '4.0 326 pixel density',
        contraindications: '1480 mAh',
        side_effect: '16GB, 32GB and RAM 1 GB'
    },
    {
        id: '4',
        categoryId: '5',
        name: 'Бахилы Стандарт, в инд.уп. 1 пара',
        description:
            'Применяются для защиты помещений от грязи и микробов. \n' +
            'В местах с высокой проходимостью, где людям необходимо снимать обувь, что небезопасно в плане гигиены, применяются для защиты ног человека.',
        price: 5,
        image: '/uploads/7.jpg',
        manufacturer: 'Медсервис Плюс, Россия',
        substance: '',
        shelf_life: 'Длинный срок',
        pharmachologic: '1',
        effect: '',
        indications: '',
        contraindications: '',
        side_effect: ''
    },
    {
        id: '5',
        categoryId: '3',
        name: 'Aussie Repair Miracle Шампунь, 300 мл',
        description:
            'Не теряй надежду на спасение своих волос. Шампунь Aussie Repair Miracle превратит сухие поврежденные волосы в мягкие, блестящие и сияющие. Попробуй всю коллекцию средств с ультрапитательным маслом австралийских семян жожоба: шампунь, бальзам-ополаскиватель и реконструктор 3 Minute Miracle.',
        price: 319,
        image: '/uploads/4.jpeg',
        manufacturer: 'Проктер энд Гэмбл, США',
        substance: '',
        shelf_life: 'Длинный срок',
        pharmachologic: '',
        effect: 'Шампунь прекрасно восстанавливает и помогает превратить сухие поврежденные волосы в мягкие, густые и сияющие. С маслом австралийских семян жожоба, способных творить настоящие чудеса.',
        indications: '',
        contraindications: '1',
        side_effect: ''
    },
    {
        id: '6',
        categoryId: '2',
        name: 'Нао Брейн таблетки покрыт. об., 2 шт.',
        description:
            'БАД. Не является лекарственным средством',
        price: 250,
        image: '/uploads/6.jpeg',
        manufacturer: 'R-Pharm Germany GmbH, Германия',
        substance: 'Экстракт листьев чая матча (Matcha)',
        shelf_life: 'Длинный срок',
        pharmachologic: '',
        effect: '',
        indications: '',
        contraindications: '',
        side_effect: ''
    }
]
